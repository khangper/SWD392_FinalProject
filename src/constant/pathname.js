export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
