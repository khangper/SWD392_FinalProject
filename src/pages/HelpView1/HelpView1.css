.HelpView-grid-container{
    display: grid;
  grid-template-columns: 1fr 3fr; /* This replaces col-lg-3 and col-lg-9 */
  gap: 10px; /* Adjust the gap as needed */
  padding: 80px 275px 100px;

}

.HelpView1-grid-container {
    margin-bottom: -97px;
    margin-top: 44px;
    display: grid;
    grid-template-columns: repeat(12, 1fr);
    padding: 0px 308px;
    gap: 20px;
}

.HelpView-title125 {
    grid-column: span 12;
    display: flex;
    justify-content: space-between;
    align-items: center;
}

.HelpView-titleleft, .explore-container {
    display: flex;
    align-items: center;
}

.HelpView-title126 {
    grid-column: span 12;
}

.HelpView-breadcrumb {
    display: flex;
    list-style: none;
    padding: 0;
    margin: 0;
}

.HelpView-breadcrumb-item {
    margin-right: 10px;
}

.HelpView-breadcrumb-item a {
    text-decoration: none;
    color: inherit;
}




.HelpView-grid-item{
   
    padding: 20px; /* For demonstration purposes */
}

.HelpView-fcrse_3 {
    background: #fff;
    width: 100%;
    float: left;
    padding: 0;
    border-radius: 3px;
	margin-bottom: 30px;
    border: 1px solid #efefef;
    transition: all .2s ease-in-out;
}

.HelpView-fcrse_3 h4 {
    font-size: 16px;
    color: #333;
	float: left;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 0;	
}

.HelpView-frc123 {
	margin-top: 13px !important;
}

.HelpView-cater_ttle{
    float: left;
	width: 100%;
	padding: 15px 20px;
	border-bottom: 1px solid #eafbff;
}

.HelpView-allcate15{
    float: left;
    width: 100%;
}

.HelpView-allcate15 li {
    display: block;
    float: left;
    width: 100%;
}

.HelpView-ct_item {
    float: left;
    width: 100%;
    padding: 10px 20px;
    font-size: 14px;
    font-family: 'Roboto', sans-serif;
    color: #686f7a;
    text-align: left;
    display: flex;
  align-items: center; /* Align items vertically */
  text-decoration: none; /* Remove underline */
  color: inherit; /* Inherit the color from the parent */
}

.HelpView-icon {
    width: 10%;
    margin-right: 8px; /* Adjust the spacing as needed */
  }

.HelpView-ct_item.active {
    font-weight: 600;
	color: #333;
}

.HelpView-ct_item i {
    margin-right: 10px;
    font-size: 120%;
}

.HelpView-rqst152{
    font-size: 14px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    color: #fff;
    background: #ed2a26;
    display: block;
    float: left;
    width: 100%;
    text-align: center;
    padding: 11px;
    height: 40px;
    border-radius: 3px;
    margin-bottom: 50px;
}

.HelpView-rqst152:hover {
    color: #fff !important;
    background: #333;
}



.HelpView-vew120 h4 {
	font-size: 18px;
    font-weight: 500;
    font-family: 'Roboto', sans-serif;
    margin-bottom: 13px !important;
	text-align: left;
	color: #333;
	line-height: 26px;
}

.HelpView-vew120 p {
	font-size: 14px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    line-height: 26px;
    color: #686f7a;
    text-align: left;
}

.HelpView-vew120 .term_link14 {
    font-size: 14px;
    font-weight: 400;
    font-family: 'Open Sans', sans-serif;
    line-height: 26px;
    color: #333;
    text-align: left;
}

.HelpView-vew120 .term_link14:hover {
    color: #333 !important;
	text-decoration: underline !important;
}

.mt-35{
    margin-top: 35px;

}

.mb-30{
    margin-bottom: 30px;

}

.HelpView-ntt125 li {
	padding-bottom: 5px;
}

.HelpView-ntt125 li:last-child {
	padding-bottom: 0;
}

.HelpView-mtl145 li {
    list-style: decimal;
    margin-left: 30px;
	font-family: 'Roboto', sans-serif;
}

.HelpView-rpt100 {
    display: flex;
    align-items: baseline;
}

.HelpView-rpt100 span {
    font-size: 14px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    color: #333;
    line-height: 24px;
	margin-right: 10px;
}

.mb-20{
margin-bottom:20px ;
}



.HelpView-radio--group-inline-container .HelpView-radio-item {
    display: inline-block;
	margin-right: 10px;
}



.HelpView-radio--group-inline-container .HelpView-radio-item input[type="radio"] + .radio-label:before {
	content: '';
	background: #f7f7f7;
	border-radius: 100%;
	border: 1px solid #b4b4b4;
	display: inline-block;
	width: 1em;
	height: 1em;
	top: 0px;
	position: relative;
	margin-right: 10px;
	vertical-align: top;
	cursor: pointer;
	text-align: center;
	-webkit-transition: all 250ms ease;
	transition: all 250ms ease;
}

.HelpView-radio--group-inline-container .HelpView-radio-item input[type="radio"]:checked + .radio-label:before {
	background-color: #ed2a26;
	box-shadow: inset 0 0 0 2px #f4f4f4;
}

.HelpView-radio--group-inline-container .HelpView-radio-item input[type="radio"]:focus + .radio-label:before {
	outline: none;
	border-color: #0E8A86;
}

.HelpView-radio--group-inline-container .HelpView-radio-item input[type="radio"]:disabled + .radio-label:before {
	box-shadow: inset 0 0 0 4px #f4f4f4;
	border-color: #b4b4b4;
	background: #b4b4b4;
}

.HelpView-radio--group-inline-container .HelpView-radio-item input[type="radio"] + .radio-label:empty:before {
	margin-right: 0;
}

.HelpView-radio-label{
    font-size: 14px;
    font-weight: 400;
    font-family: 'Roboto', sans-serif;
    line-height: 12px;
	color: #333;
	margin-bottom: 0;
	cursor: pointer;
}

.HelpView-radio--group-inline-container .HelpView-radio-item input[type="radio"] + .radio-label:before {
    content: '';
    background: #f7f7f7;
    border-radius: 100%;
    border: 1px solid #b4b4b4;
    display: inline-block;
    width: 1em;
    height: 1em;
    top: 0px;
    position: relative;
    margin-right: 10px;
    vertical-align: top;
    cursor: pointer;
    text-align: center;
    -webkit-transition: all 250ms ease;
    transition: all 250ms ease;
}