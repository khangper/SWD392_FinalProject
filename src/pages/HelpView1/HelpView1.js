import React from 'react'
import Header from '../../components/Header-paidmember/Header'
import "./HelpView1.css";
import Footer from '../../components/Footer/Footer'
import search_icon from '..//../assets/search.png';
import Payment from '..//../assets/payment.png';
import tablerow from '..//../assets/tablerow.png';
import helpdestop from '..//../assets/helpdestop.png';
import rulerhelp from '..//../assets/ruler-help.png';
import trade from '..//../assets/trade.png';
import bookhelp from '..//../assets/book-help.png';



function HelpView1() {
  return (
    <div className="HelpView1">
        <Header/>

        <div className="HelpView1-grid-container">
    <div className="HelpView-title125">
        <div className="HelpView-titleleft">
            <div className="HelpView-ttl121">
                <nav aria-label="breadcrumb">
                    <ol className="HelpView-breadcrumb">
                        <li className="HelpView-breadcrumb-item"><a href="index.html">Home</a></li>
                        
                        <li className="HelpView-breadcrumb-item active" aria-current="page">/Payment</li>
                    </ol>
                </nav>
            </div>
        </div>
        <div className="explore-container">
            <section className="search-box">
        <img src={search_icon} alt="Search Icon" className="search-icon" />
        <input
          type="text"
          placeholder="Search Tutors"
        />
      </section>
            </div>
    </div>
    <div className="HelpView-title126">
        <h2>Promote Your Course With Coupons and Referral Links</h2>
    </div>
</div>



        {/* Body HelpView1 */}

        <div className="HelpView-grid-container">
            <div className="HelpView-grid-item">

            <div className="HelpView-fcrse_3 HelpView-frc123">
  <div className="HelpView-cater_ttle">
    <h4>Instructor Topics</h4>
  </div>
  <ul className="HelpView-allcate15">
    <li><a href="#" className="HelpView-ct_item"><img src={Payment} alt="Search Icon" className="HelpView-icon" />Payments</a></li>
    <li><a href="#" className="HelpView-ct_item active"><img src={tablerow} alt="Search Icon" className="HelpView-icon" />Selling & Promotion</a></li>
    <li><a href="#" className="HelpView-ct_item"><img src={helpdestop} alt="Search Icon" className="HelpView-icon" />Quality Standards</a></li>
    <li><a href="#" className="HelpView-ct_item"><img src={rulerhelp} alt="Search Icon" className="HelpView-icon" />Course Building</a></li>
    <li><a href="#" className="HelpView-ct_item"><img src={trade} alt="Search Icon" className="HelpView-icon" />Course Management</a></li>
    <li><a href="#" className="HelpView-ct_item"><img src={bookhelp} alt="Search Icon" className="HelpView-icon" />Trust & Safety</a></li>
  </ul>
</div>
<div className="HelpView-fcrse_3">
  <div className="HelpView-cater_ttle">
    <h4>Related Articles</h4>
  </div>
  <ul className="HelpView-allcate15">
    <li><a href="#" className="HelpView-ct_item">Instructor Revenue Share</a></li>
    <li><a href="#" className="HelpView-ct_item">Coupons & Course Referral Links: Rules and Guidelines</a></li>
    <li><a href="#" className="HelpView-ct_item">Rules and Guidelines</a></li>
    <li><a href="#" className="HelpView-ct_item">Instructor Promotional Agreements and Cursus Deals</a></li>
    <li><a href="#" className="HelpView-ct_item">Cursus Coupons: FAQ</a></li>
    <li><a href="#" className="HelpView-ct_item">How to Select Your Payout Method And Become a Premium Instructor</a></li>
  </ul>
</div>

<a href="#" className="HelpView-rqst152">Submit A Request</a>


            </div>
            
            <div className="HelpView-grid-item">
            <div className="HelpView-vew120 HelpView-frc123">
							<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur volutpat maximus pellentesque. Integer sem enim, luctus at nibh at, condimentum sagittis sapien. Sed tempus ipsum erat, sit amet efficitur velit interdum eu. Vestibulum hendrerit id dolor eu scelerisque. Phasellus ex dui, consequat nec feugiat eu, dapibus eget ante. Sed sodales interdum dui, at euismod mi feugiat hendrerit. Suspendisse auctor libero in tempor mollis. Nulla et dolor velit. Aliquam sit amet luctus quam.</p>
						</div>
						<div className="HelpView-vew120 mt-35 mb-30">			
							<h4>Course referral links</h4>
							<p>Nam a egestas libero, eget eleifend turpis. Sed id ipsum a ipsum aliquam laoreet sit amet sit amet nibh. Proin dapibus, libero sed posuere rhoncus, orci mi cursus enim, at accumsan eros massa lacinia mi. Nunc eget finibus felis, volutpat malesuada sem. Aliquam ac nisl pellentesque, varius neque sit amet, porttitor nunc. Nullam elit tellus, dapibus non eleifend sed, hendrerit eget velit. Aliquam ut felis dictum, tincidunt magna vitae, aliquam massa. In porttitor tristique quam, non dignissim sapien pharetra ultrices. Cras non ante non velit mollis mollis. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.</p>
							<p className="mt-4">Etiam lobortis dictum tincidunt. Interdum et malesuada fames ac ante ipsum primis in faucibus. Suspendisse ultricies efficitur dui, suscipit tempus elit condimentum quis. Duis sed vestibulum tortor, eget cursus odio.</p>
						</div>
						<div className="HelpView-vew120 mt-35 mb-30">			
							<h4>How to create a coupon</h4>
							<ol className="HelpView-ntt125 mtl145">
								<li><p>Navigate to the <strong>course Promotions page</strong></p></li>								
								<li><p>Click <strong>Create new coupon</strong></p></li>								
								<li><p><strong>Select the coupon type</strong></p></li>								
								<li><p>If applicable, choose your coupon price</p></li>								
								<li><p>Edit the coupon code if you wish (the coupon code must be between 6 and 20 characters, and can only contain alphanumeric characters (A-Z, 0-9) , periods ("."), dashes ("-") or underscores ("_").</p></li>								
								<li><p>Click <strong>Review coupon</strong> to double check the couponâ€™s details</p></li>								
								<li><p>Click <strong>Create coupon</strong></p></li>								
							</ol>
						</div>
						<div className="HelpView-vew120 mt-35 mb-30">			
							<h4>How to share your coupon code</h4>
							<p>Morbi lectus nunc, lacinia ut consequat a, semper vel erat. Duis ut lacus nec dui sodales mattis. Mauris tellus dolor, pulvinar sit amet pretium a, malesuada sed tellus. Aliquam ultrices elit neque, quis lacinia ex porttitor non. Donec ac iaculis turpis. Nulla lacinia enim quis orci aliquam, non cursus urna pellentesque. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Phasellus in mi a nisi auctor interdum. Vivamus faucibus magna sed elit interdum consequat. Vestibulum eu tortor vel ante feugiat faucibus quis et urna. Quisque interdum ac enim eu tempus. Fusce viverra, lectus egestas tincidunt cursus, tortor sapien convallis metus, vitae auctor risus metus vel nisi. Aenean dapibus bibendum mauris ut iaculis.</p>
						</div>
						<div className="HelpView-vew120 mt-35 mb-30">			
							<h4>How to deactivate a coupon</h4>
							<p>Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.</p>
						</div>
						<div className="HelpView-rpt100 mt-35 mb-20">
							<span>Was this review helpful?</span>
							<div className="HelpView-radio--group-inline-container">
								<div className="HelpView-radio-item">
                                <input id="radio-1" name="radio" type="radio"/>
                                <label htmlFor="radio-1" className="HelpView-radio-label">Yes</label>
								</div>
								<div className="HelpView-radio-item">
									<input id="radio-2" name="radio" type="radio"/>
									<label htmlFor="radio-2" className="HelpView-radio-label">No</label>
								</div>
							</div>
						</div>

            </div>

        </div>

        <Footer/>


      
    </div>
  )
}

export default HelpView1
