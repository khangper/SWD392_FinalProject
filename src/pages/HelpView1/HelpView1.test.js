import React from "react";
import { render, screen } from "@testing-library/react";
import HelpView1 from "./HelpView1";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("HelpView1", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the HelpView1 component", async () => {
    const { container } = renderWithProviders(<HelpView1 />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });

  it("displays the breadcrumb correctly", () => {
    renderWithProviders(<HelpView1 />);
    const breadcrumb = screen.getByText("/Payment");
    expect(breadcrumb).toBeInTheDocument();
  });

  it("displays the main title correctly", () => {
    renderWithProviders(<HelpView1 />);
    const mainTitle = screen.getByText(
      "Promote Your Course With Coupons and Referral Links"
    );
    expect(mainTitle).toBeInTheDocument();
  });

  it("displays the search input box correctly", () => {
    renderWithProviders(<HelpView1 />);
    const searchInput = screen.getByPlaceholderText("Search Tutors");
    expect(searchInput).toBeInTheDocument();
  });

  it("displays the 'Submit A Request' link correctly", () => {
    renderWithProviders(<HelpView1 />);
    const submitRequestLink = screen.getByText("Submit A Request");
    expect(submitRequestLink).toBeInTheDocument();
    expect(submitRequestLink.closest("a")).toHaveAttribute("href", "#");
  });
});
