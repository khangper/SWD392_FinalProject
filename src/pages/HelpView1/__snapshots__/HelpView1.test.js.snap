// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpView1 renders the HelpView1 component 1`] = `
<div>
  <div
    class="HelpView1"
  >
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <div
      class="HelpView1-grid-container"
    >
      <div
        class="HelpView-title125"
      >
        <div
          class="HelpView-titleleft"
        >
          <div
            class="HelpView-ttl121"
          >
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="HelpView-breadcrumb"
              >
                <li
                  class="HelpView-breadcrumb-item"
                >
                  <a
                    href="index.html"
                  >
                    Home
                  </a>
                </li>
                <li
                  aria-current="page"
                  class="HelpView-breadcrumb-item active"
                >
                  /Payment
                </li>
              </ol>
            </nav>
          </div>
        </div>
        <div
          class="explore-container"
        >
          <section
            class="search-box"
          >
            <img
              alt="Search Icon"
              class="search-icon"
              src="search.png"
            />
            <input
              placeholder="Search Tutors"
              type="text"
            />
          </section>
        </div>
      </div>
      <div
        class="HelpView-title126"
      >
        <h2>
          Promote Your Course With Coupons and Referral Links
        </h2>
      </div>
    </div>
    <div
      class="HelpView-grid-container"
    >
      <div
        class="HelpView-grid-item"
      >
        <div
          class="HelpView-fcrse_3 HelpView-frc123"
        >
          <div
            class="HelpView-cater_ttle"
          >
            <h4>
              Instructor Topics
            </h4>
          </div>
          <ul
            class="HelpView-allcate15"
          >
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                <img
                  alt="Search Icon"
                  class="HelpView-icon"
                  src="payment.png"
                />
                Payments
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item active"
                href="#"
              >
                <img
                  alt="Search Icon"
                  class="HelpView-icon"
                  src="tablerow.png"
                />
                Selling & Promotion
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                <img
                  alt="Search Icon"
                  class="HelpView-icon"
                  src="helpdestop.png"
                />
                Quality Standards
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                <img
                  alt="Search Icon"
                  class="HelpView-icon"
                  src="ruler-help.png"
                />
                Course Building
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                <img
                  alt="Search Icon"
                  class="HelpView-icon"
                  src="trade.png"
                />
                Course Management
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                <img
                  alt="Search Icon"
                  class="HelpView-icon"
                  src="book-help.png"
                />
                Trust & Safety
              </a>
            </li>
          </ul>
        </div>
        <div
          class="HelpView-fcrse_3"
        >
          <div
            class="HelpView-cater_ttle"
          >
            <h4>
              Related Articles
            </h4>
          </div>
          <ul
            class="HelpView-allcate15"
          >
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                Instructor Revenue Share
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                Coupons & Course Referral Links: Rules and Guidelines
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                Rules and Guidelines
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                Instructor Promotional Agreements and Cursus Deals
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                Cursus Coupons: FAQ
              </a>
            </li>
            <li>
              <a
                class="HelpView-ct_item"
                href="#"
              >
                How to Select Your Payout Method And Become a Premium Instructor
              </a>
            </li>
          </ul>
        </div>
        <a
          class="HelpView-rqst152"
          href="#"
        >
          Submit A Request
        </a>
      </div>
      <div
        class="HelpView-grid-item"
      >
        <div
          class="HelpView-vew120 HelpView-frc123"
        >
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur volutpat maximus pellentesque. Integer sem enim, luctus at nibh at, condimentum sagittis sapien. Sed tempus ipsum erat, sit amet efficitur velit interdum eu. Vestibulum hendrerit id dolor eu scelerisque. Phasellus ex dui, consequat nec feugiat eu, dapibus eget ante. Sed sodales interdum dui, at euismod mi feugiat hendrerit. Suspendisse auctor libero in tempor mollis. Nulla et dolor velit. Aliquam sit amet luctus quam.
          </p>
        </div>
        <div
          class="HelpView-vew120 mt-35 mb-30"
        >
          <h4>
            Course referral links
          </h4>
          <p>
            Nam a egestas libero, eget eleifend turpis. Sed id ipsum a ipsum aliquam laoreet sit amet sit amet nibh. Proin dapibus, libero sed posuere rhoncus, orci mi cursus enim, at accumsan eros massa lacinia mi. Nunc eget finibus felis, volutpat malesuada sem. Aliquam ac nisl pellentesque, varius neque sit amet, porttitor nunc. Nullam elit tellus, dapibus non eleifend sed, hendrerit eget velit. Aliquam ut felis dictum, tincidunt magna vitae, aliquam massa. In porttitor tristique quam, non dignissim sapien pharetra ultrices. Cras non ante non velit mollis mollis. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.
          </p>
          <p
            class="mt-4"
          >
            Etiam lobortis dictum tincidunt. Interdum et malesuada fames ac ante ipsum primis in faucibus. Suspendisse ultricies efficitur dui, suscipit tempus elit condimentum quis. Duis sed vestibulum tortor, eget cursus odio.
          </p>
        </div>
        <div
          class="HelpView-vew120 mt-35 mb-30"
        >
          <h4>
            How to create a coupon
          </h4>
          <ol
            class="HelpView-ntt125 mtl145"
          >
            <li>
              <p>
                Navigate to the 
                <strong>
                  course Promotions page
                </strong>
              </p>
            </li>
            <li>
              <p>
                Click 
                <strong>
                  Create new coupon
                </strong>
              </p>
            </li>
            <li>
              <p>
                <strong>
                  Select the coupon type
                </strong>
              </p>
            </li>
            <li>
              <p>
                If applicable, choose your coupon price
              </p>
            </li>
            <li>
              <p>
                Edit the coupon code if you wish (the coupon code must be between 6 and 20 characters, and can only contain alphanumeric characters (A-Z, 0-9) , periods ("."), dashes ("-") or underscores ("_").
              </p>
            </li>
            <li>
              <p>
                Click 
                <strong>
                  Review coupon
                </strong>
                 to double check the couponâ€™s details
              </p>
            </li>
            <li>
              <p>
                Click 
                <strong>
                  Create coupon
                </strong>
              </p>
            </li>
          </ol>
        </div>
        <div
          class="HelpView-vew120 mt-35 mb-30"
        >
          <h4>
            How to share your coupon code
          </h4>
          <p>
            Morbi lectus nunc, lacinia ut consequat a, semper vel erat. Duis ut lacus nec dui sodales mattis. Mauris tellus dolor, pulvinar sit amet pretium a, malesuada sed tellus. Aliquam ultrices elit neque, quis lacinia ex porttitor non. Donec ac iaculis turpis. Nulla lacinia enim quis orci aliquam, non cursus urna pellentesque. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Phasellus in mi a nisi auctor interdum. Vivamus faucibus magna sed elit interdum consequat. Vestibulum eu tortor vel ante feugiat faucibus quis et urna. Quisque interdum ac enim eu tempus. Fusce viverra, lectus egestas tincidunt cursus, tortor sapien convallis metus, vitae auctor risus metus vel nisi. Aenean dapibus bibendum mauris ut iaculis.
          </p>
        </div>
        <div
          class="HelpView-vew120 mt-35 mb-30"
        >
          <h4>
            How to deactivate a coupon
          </h4>
          <p>
            Quisque et bibendum urna, eget consequat sapien. Integer sed condimentum nibh. Integer id neque tristique, lobortis massa ac, dapibus nibh. Donec nulla odio, porttitor ac rutrum eget, volutpat a velit. Curabitur et enim quis diam congue dictum et vitae dui. Nulla tortor orci, luctus a pretium vel, ultrices porta nisl.
          </p>
        </div>
        <div
          class="HelpView-rpt100 mt-35 mb-20"
        >
          <span>
            Was this review helpful?
          </span>
          <div
            class="HelpView-radio--group-inline-container"
          >
            <div
              class="HelpView-radio-item"
            >
              <input
                id="radio-1"
                name="radio"
                type="radio"
              />
              <label
                class="HelpView-radio-label"
                for="radio-1"
              >
                Yes
              </label>
            </div>
            <div
              class="HelpView-radio-item"
            >
              <input
                id="radio-2"
                name="radio"
                type="radio"
              />
              <label
                class="HelpView-radio-label"
                for="radio-2"
              >
                No
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                â–¼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          Â© 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
